src/modules/user/controllers/controllerUpdate.js

```diff
@@ -104,8 +104,8 @@ export const handleUpdate = async function (req, res) {
       }
     }
 
-    if (skills !== undefined) {
-      details.skills = skills.split(",");
+    if (Array.isArray(skills)) {
+      details.skills = skills;
     }
```

Comment:
At a quick glance, I think this might be a better setup as this is the update controller, so the payload will vary, but it might be wise to add some feedback for incorrect formats.
```suggestion
    if (skills !== undefined) {
        if (Array.isArray(skills)) {
           details.skills = skills;
       } else {
         return res.status(400).json({ message: "skills must be an array." });
       }
   }
```

Looking at it, might be wise to check through the entire payload, and give feedback on all errors in the payload rather than just 1 at a time.

Reply:
Thanks for the feedback, added your changes and will see if i can  look at the other part tomorrow.